Source/APIClient/Request.swift

```diff
@@ -34,15 +36,22 @@ public protocol Request {
 
 public extension Request {
 
+    var contentType: String {
+        switch self {
+        default:
+            return "application/json"
+        }
+    }
```

Comment:
Default implementation seems fine. There are a ton of theoretical options for this value, but we're already assuming json in other parts of the library so I'm ok with just using a single string value.